packages/adapter-vercel/index.js

```diff
@@ -412,8 +414,22 @@ function static_vercel_config(builder) {
 		overrides[page.file] = { path: overrides_path };
 	}
 
+	/** @type {Record<string, any> | undefined} */
+	let images = undefined;
+	if (config.images) {
+		images = {
+			// TODO this is duplicated in @sveltejs/image -> figure out a good way to keep them in sync / make them configurable
+			sizes: [64, 96, 128, 256, 384, 640, 750, 828, 1080, 1200, 1920, 2048, 3840],
+			domains: Array.isArray(config.images?.domains) ? config.images.domains : [],
+			// TODO should we expose the following and some other optional options through the adapter?
+			formats: ['image/avif', 'image/webp'],
+			minimumCacheTTL: 300
+		};
+	}
```

Comment:
I think you could write this as:
```suggestion
	const images = config.images ?? {
			// TODO this is duplicated in @sveltejs/image -> figure out a good way to keep them in sync / make them configurable
			sizes: [64, 96, 128, 256, 384, 640, 750, 828, 1080, 1200, 1920, 2048, 3840],
			domains: Array.isArray(config.images?.domains) ? config.images.domains : [],
			// TODO should we expose the following and some other optional options through the adapter?
			formats: ['image/avif', 'image/webp'],
			minimumCacheTTL: 300
		};
```

Reply:
No, `config.images` is a `boolean` indicating you want to opt into Vercel's image optimization, which then sets the config.